components/ModDashboardLeftNavbar.vue

```diff
@@ -1,64 +1,56 @@
 <template>
     <div>
-        <select v-model="selectedDashboardView" class="font-bold border-b border-black mb-2">
-            <option :key="dashboardViewOption.value" :value="dashboardViewOption.value"
-                v-for="(dashboardViewOption, index) in modDashboardViewOptions">
-                {{ $t(dashboardViewOption.displayText) }}
+        <select @change="updateSelectedDashboardView" class="font-semibold p-1 border-b border-black mb-2">
+            <option value="facilities">
+                {{ $t("modDashboardLeftNav.facilities") }}
+            </option>
+            <option value="healthcareProfessionals">
+                {{ $t("modDashboardLeftNav.healthcareProfessionals") }}
             </option>
         </select>
         <div class="flex flex-col">
-            <button @click="updateSubmissionListViewState(submissionListEntry.value)"
-                class="flex flex-row items-center text-start p-1" :key="submissionListEntry.value"
-                :class="{ 'border-b border-gray-300': index !== submissionListViewOptions.length - 1 }"
-                v-for="(submissionListEntry, index) in submissionListViewOptions ">
-                <div class="flex items-center justify-center w-1/5">
-                    <component :is="submissionListEntry.svgIcon" class="h-5" />
-                </div>
-                {{ $t(submissionListEntry.displayText) }} ({{ submissionListEntry.count }})
-            </button>
+            <div class="flex flex-row w-4/5 justify-start items-center border-b-2 border-slate-200">
```

Comment:
```suggestion
            <div class="flex w-4/5 justify-start items-center border-b-2 border-slate-200">
```

can remove redundant `flex-row`